<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgShowImage</class>
 <widget class="QDialog" name="dlgShowImage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>986</width>
    <height>703</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>620</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="HHalconLabelAuto" name="lblDisplay">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>811</width>
     <height>681</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="btnSrc">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>30</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Source</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="btnResult">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>110</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Result</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExport">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>540</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HHalconLabelAuto</class>
   <extends>QLabel</extends>
   <header location="global">hhalconlabelauto.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>btnOK</sender>
   <signal>clicked()</signal>
   <receiver>dlgShowImage</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>905</x>
     <y>650</y>
    </hint>
    <hint type="destinationlabel">
     <x>492</x>
     <y>351</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
